# This message describes the names and identification numbers of
# all entities in the system.
# Header
imc_ros_interface/Header header
# Operation to perform, 0 = Report, 1 = Query
std_msgs/UInt8 op
std_msgs/String list 
